src/python/library/requirements/requirements_grpc.txt

```diff
@@ -24,7 +24,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-protobuf>=3.5.0
+protobuf>=3.5.0,<3.20
```

Comment:
Can you add a comment mentioning the jira ticket for upgrading in the future?

Reply:
Sure, done.